frontend/src/components/common/LogRecordsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { LogRecord } from "../../types/LogRecordTypes";
+
+type Props = {
+  logRecords: LogRecord[];
+};
+
+const LogRecordsTable = ({ logRecords }: Props): React.ReactElement => {
+  return (
+    <div>
+      <TableContainer marginTop="12px" height="70vh" overflowY="scroll">
+        <Table
+          variant="simple"
+          style={{ minHeight: "400px", verticalAlign: "middle" }}
+        >
+          <Thead className="table-header">
+            <Tr>
+              <Th>Date</Th>
+              <Th>Time</Th>
+              <Th>Resident</Th>
+              <Th>Note</Th>
+              <Th>Employee</Th>
+              <Th>Attn To</Th>
+            </Tr>
+          </Thead>
+
+          <Tbody>
+            {/* TODO: replace mockRecords with logRecords */}
+            {logRecords.map((record) => {
+              // TODO: Investigate alternative methods for date storage + creation
+              const dateObj = new Date(record.datetime);
+
+              return (
+                <Tr key={record.logId} style={{ verticalAlign: "middle" }}>
+                  <Td width="5%">
+                    {`${dateObj.getMonth()} ${dateObj.getDate()} ${dateObj.getFullYear()}`}
```

Comment:
Would it be possible to show an abbreviated form for the month instead of the number? For example, instead of "9 2 2023", we would see Sept 9th, 2023.

Reply:
Resolved ; used the following const array for the months. Let me know if you want any of these to be changed:

```
const MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
```

Reply:
Following up on this ; the year is **not** included in the designs ; this was accidentally included by us. On that note though, is the year something we want to include?

Reply:
yeah, might be confusing if you have two logs that are both jan 18th (but one is 2023, the other 2024)

Reply:
will do

Reply:
the months are correct, but the year isn't showing: 
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/7f94c1e2-b22f-4cc3-a065-62f2571f7b97)